models/event.ts

```diff
@@ -0,0 +1,8 @@
+interface Event {
+    name: string,
+    hostclub: string    //club name. THIS NEEDS TO BE THE SAME AS THE CLUB ID IN THE DATABASE. 
```

Comment:
Per the comment, does anything need to change here prior to merging?

Reply:
We can remove the comment. It looks like the club name is registered as it's ID.

Reply:
Sorry, this is an event. Not a club. 
I don't think we should make use of functions that work with events yet